.github/workflows/python-demos.yml

```diff
@@ -46,18 +47,30 @@ jobs:
     - name: Pre-build dependencies
       run: python -m pip install --upgrade pip
 
+    # ************** REMOVE AFTER RELEASE ********************
+    - name: Build binding
+      run: |
+        pip install wheel setuptools && cd ../../binding/python && python3 setup.py sdist bdist_wheel && pip install --force-reinstall dist/pvrhino-4.0.0-py3-none-any.whl
+    # ********************************************************
+
     - name: Install dependencies
       run: pip install -r requirements.txt
 
     - name: Test
-      run: python rhino_demo_file.py --access_key ${{secrets.PV_VALID_ACCESS_KEY}} --wav_path ../../resources/audio_samples/test_within_context.wav --context_path ../../resources/contexts/${{ matrix.platform }}/coffee_maker_${{ matrix.platform }}.rhn
+      run: >
+        python rhino_demo_file.py 
+        --access_key ${{secrets.PV_VALID_ACCESS_KEY}}
+        --device ${{ matrix.device }}
+        --wav_path ../../resources/audio_samples/test_within_context.wav 
+        --context_path ../../resources/contexts/${{ matrix.platform }}/coffee_maker_${{ matrix.platform }}.rhn
 
   build-self-hosted:
     runs-on: ${{ matrix.machine }}
 
     strategy:
       matrix:
-        machine: [rpi3-32, rpi3-64, rpi4-32, rpi4-64, rpi5-64, pv-windows-arm64]
+        device: [ best ]
+        machine: [ rpi3-32, rpi3-64, rpi4-32, rpi4-64, rpi5-64, pv-windows-arm64 ]
```

Comment:
for desktop SDKs, we've also been adding the GPU runners `pv-windows`, `pv-ios` and `pv-linux` to run the demos and tests with device `gpu`. Check out Eagle for what that looks like